src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -286,11 +286,15 @@ void ModeSet(chip::EndpointId endpoint, chip::BitMask<Mode> & newMode)
         newMode.Clear(Mode::kCalibrationMode);
     }
 
-    if (oldMode != newMode)
+    if (oldMode.Raw() != newMode.Raw())
```

Comment:
Why this change?

In general, all the actual code changes do not belong hidden in a "bump submodule revision" PR.  They should have their own PRs, with clear descriptions, which can then get reviewed by the people famiiar with the relevant code areas.

Reply:
clang-tidy: this was an implicit conversio nto enum value and the enum value was out of range. We need an actual comparison operator or compare something that is of valid types.

Reply:
Do we not have an operator!= (or operator==) on BitFlags/BitMask?  If not, we should.